commerce-media-impl/src/main/java/com/liferay/commerce/media/internal/DefaultCommerceMediaResolver.java

```diff
@@ -143,7 +143,7 @@ public String getUrl(
 		}
 		else {
 			cpAttachmentFileEntry =
-				_cpAttachmentFileEntryLocalService.fetchCPAttachmentFileEntry(
+				_cpAttachmentFileEntryService.fetchCPAttachmentFileEntry(
```

Comment:
check secure pattern implemented in ExpandoBridge

Reply:
@ethanbustad  in portal secure is used differently ExpandoBridgeImpl.addAttribute

Reply:
It still seems flipped to me. Wouldn't be more clear to use a variable like `checkPermissions` or something like that? Cause at this point if someone sees `secure` as a parameter theres a 50% chance they'll think of it either way.

Reply:
for reference, https://www.merriam-webster.com/dictionary/secure 
`1a: free from danger`
which would imply if (secure) then use local service ;)